src/features/presentation/layouts/presentation/presentation.layout.html

```diff
@@ -0,0 +1,141 @@
+<div style="position: relative">
+  <img src="../../../../../../assets/img/presentation/fond-landing.svg" class="img-fluid" alt="image_fond_landing" />
```

Comment:
```suggestion
  <img src="/assets/img/presentation/fond-landing.svg" class="img-fluid" alt="" />
```
L'attribut `alt` est lu à haute voix par les outils à disposition des personnes ayant des déficiences visuelles, il faut donc prendre autant de soin pour son contenu que n'importe quel autre texte affiché à l'écran.
Si'il n'est pas pertinent de décrire l'image (par exemple une image décorative qui n'apporte pas d'information), il faut mettre `alt=""`